pegasus/sites.v3/hourofcode.com/views/signup_form.haml

```diff
@@ -41,17 +41,19 @@
           %input#hoc-entire-school-flag{name: "entire_school_flag_b", type: "checkbox", value: "1"}/
           =hoc_s(:signup_event_entire_school_label)
 
-    - if @country == 'us'
-      .form-group
-        .checkbox
-          %label
-            %input#hoc-send-posters-flag{name: "send_posters_flag_b", type: "checkbox", value: "1"}/
-            =hoc_s(:signup_send_posters_flag_label)
-
-      .form-group{style: "display: none;"}
-        %label.control-label{for: "hoc-send-posters-address"}= hoc_s(:signup_send_posters_address_label)
-        %div
-          %textarea#hoc-send-posters-address.form-control{name: "send_posters_address_s", placeholder: hoc_s(:signup_send_posters_address_placeholder), style: "height: 5em;"}
+    / Hide the posters checkbox when # of requests hits our quota.
+    / Show the posters checkbox again when signup form opens for next Hour of Code.
+    / - if @country == 'us'
```

Comment:
Possible alternative approaches: 

- delete the unused code
- comment out using a single `-#` at the top of the indentation hierarchy
- disable with `if false` at top of the code so it doesn't run but is also not commented out.

No strong preference, but I'm not a huge fan of commenting out each individual line.